buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -0,0 +1,69 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.util.VersionNumber
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Elements
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+open class UpdateEapBuildNumber : DefaultTask() {
+
+    @Internal
+    var intellijSnapshotsUrl: String = "https://www.jetbrains.com/intellij-repository/snapshots/"
+
+    @Internal
```

Comment:
Hmm if not marked as `@Internal`... will it automatically be interpreted as `@Input` or `@Output`? how does `@Internal` var differ from an annotation-less var?

Reply:
If no annotation is found gradle throws an error - [Gradle Docs](https://docs.gradle.org/7.4.2/userguide/validation_problems.html#missing_annotation)

Reply:
Cool picture description from [Gradle Docs](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:up_to_date_checks)
<img width="745" alt="image" src="https://user-images.githubusercontent.com/61919032/181498344-624dac00-9bb7-408f-b82b-941fc8927a91.png">